thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/detection/AnomalyFeedbackReason.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.spi.detection;
+
+public enum AnomalyFeedbackReason {
```

Comment:
is this list set in stone? has it been validated? 
Should this really be a plain enum? I fear users will ask for custom values.
Should we plan a free text field for OTHER? 

Eg JET sometimes have DDOS attacks, so I think something like CYBER_ATTACK would be relevant. But maybe it's not.

Reply:
User will be provided a multi-checkbox for this field and they are allowed to check multiple reasons (aka causes) when editing the feedback. So it's 1:n relation between feedback and reasons. 

You can check the screenshot in https://cortexdata.atlassian.net/browse/TE-1498

Reply:
Understood. 
> You can check the screenshot in https://cortexdata.atlassian.net/browse/TE-1498

I'm actually challenging the multi-select. 
Eg in
``` 
HOLIDAY_EFFECT
USER_ERROR
SOFTWARE_FAILURE
ENVIRONMENTAL_FACTOR
FRAUD
PLATFORM_UPGRADE
EXTERNAL_EVENT
OTHER
```

Does it really make sense to have multi-select? 
the possible values look exclusive to me.

Eg if it's a USER_ERROR then it cannot be the other possible values.

I'm fine with multi-select as long as we are aware of at least one case where it's relevant.

Reply:
Had discussion with @spham92 and we decided to support only one cause at this time. So I will make a change on this one.